frontend/packages/text-editor/src/TextList.tsx

```diff
@@ -41,14 +41,15 @@ export const TextList = ({
       <TableBody>
         {resourceRows
           .filter((row) => filterFunction(row.textKey, row.translations, searchQuery))
-          .map((row) => (
+          .map((row, index) => (
             <TextRow
               key={`${row.translations[0].lang}.${row.textKey}`}
               textId={row.textKey}
               idExists={idExists}
               textRowEntries={row.translations}
               variables={row.variables || []}
               selectedLanguages={selectedLanguages}
+              showButton={index !== 0}
```

Comment:
Can we assume that `appName` will always be the first text? App developers can edit the text files manually, they might change the order. I think it would be safer to check the textKey explicitly, f.ex.
```suggestion
              showButton={row.textKey !== 'appName'}
```

Reply:
changed and removed index paramater